include/core/parameters.h

```diff
@@ -371,7 +372,7 @@ namespace Parameters
       phase_change
     } thermal_expansion_model;
 
-    // Linear thermal conductivity parameters: k = k_A0 + k_A1 * T
+    // Linear thermal conductivity parameters : k = k_A0 + k_A1 * T
```

Comment:
```suggestion
    // Linear thermal conductivity parameters: k = k_A0 + k_A1 * T
```
no space before : in English